nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -44,7 +44,25 @@ let
     }
   ''));
 
-  configFile = pkgs.writeText "nginx.conf" ''
+  awkFormat = pkgs.writeText "awkFormat-nginx.awk" ''
```

Comment:
I usually prefer `builtins.toFile` when the file is constant, because then there's one less derivation to build.